Sources/PackageDescription/SupportedPlatforms.swift

```diff
@@ -95,23 +100,27 @@ public struct SupportedPlatform: Encodable, Equatable {
         self.version = version
     }
 
-    /// Configures the minimum deployment target version for the macOS platform.
+    /// Configure the minimum deployment target version for the macOS platform.
```

Comment:
The existing convention is using third-person present tense for a function's description. E.g. https://developer.apple.com/documentation/swift/array/1641390-remove